readme.dev.md

```diff
@@ -35,4 +37,4 @@ If you have a patch for the config generator scripts that improves the generated
 
 ### Custom swagger-codegen templates
 
-In some cases the standard swagger-codegen templates may not be suitable with the OneFS API, necessitating the use of [custom templates](https://github.com/swagger-api/swagger-codegen/wiki/Building-your-own-Templates). Some [Python templates](./swagger_templates/python) for example, have been customized for generation of the Isilon SDK. Customized templates are located in `/swagger_templates/<language>`. This naming convention allows the automated build process to locate and use the appropriate custom templates. If no custom template exists for a language, the default template will be used automatically.
\ No newline at end of file
+In some cases the standard swagger-codegen templates may not be suitable with the OneFS API, necessitating the use of [custom templates](https://github.com/swagger-api/swagger-codegen/wiki/Building-your-own-Templates). Some [Python templates](./swagger_templates/python) for example, have been customized for generation of the Isilon SDK. Customized templates are located in `/swagger_templates/<language>`. This naming convention allows the automated build process to locate and use the appropriate custom templates. If no custom template exists for a language, the default template will be used automatically.
```

Comment:
I'm not sure why this is showing up in the diff, as nothing appears to have nor should have changed in this line.